vm-systemd/network-pre.target.d/30_qubes.conf

```diff
@@ -0,0 +1,3 @@
+[Unit]
+Requires=qubes-antispoof.service
+After=qubes-antispoof.service
```

Comment:
I think it's better to add them to the `qubes-antispoof.service` instead (as `RequiredBy=` and `Before=` respectively).